### PR TITLE
Refactor: Avoid class variable for moesif config option

### PR DESCRIPTION
**Pull request by keyur9**
Refactor: Avoid class variable for moesif config option
Refactor: Convert response headers and eTag key to lowercase
Bump version to 1.3.2

### CODE REVIEW

File: Moesif.Middleware/NetCore/MoesifMiddlewareNetCore.cs [hunk 1]
```diff
@@ -23,12 +23,6 @@ public class MoesifMiddlewareNetCore
     {
         private readonly RequestDelegate _next;
 
-        public string userId;
-
-        public string companyId;
-
-        public string sessionToken;
-
         public Dictionary<string, object> metadata;
```
Comment (dgilling) [inline Moesif.Middleware/NetCore/MoesifMiddlewareNetCore.cs]: Doesn't metadata also need to be moved?

**Review by dkm199 — APPROVED** [review]
lgtm

File: Moesif.Middleware/NetCore/MoesifMiddlewareNetCore.cs (hunk 1 above)
Comment (dkm199) [reply Moesif.Middleware/NetCore/MoesifMiddlewareNetCore.cs]: yeah..talked offline..he fixed it next commit

Comment (dgilling) [reply Moesif.Middleware/NetCore/MoesifMiddlewareNetCore.cs]: ok. Thanks.

**Pull request merged**